compiler/src/main/scala/dx/compiler/DxExecutableDirectory.scala

```diff
@@ -75,13 +75,16 @@ case class DxExecutableDirectory(bundle: Bundle,
     val desc: Option[DxObjectDescribe] = Some(dxDesc)
   }
 
-  private def findExecutables(folder: Option[String]): Vector[(DxDataObject, DxObjectDescribe)] = {
+  private def findExecutables(
+      folder: Option[String] = None,
+      recurse: Boolean = false
+  ): Vector[(DxDataObject, DxObjectDescribe)] = {
     Vector("applet", "workflow")
       .flatMap { dxClass =>
         val constraints = DxFindDataObjectsConstraints(
             project = Some(project),
             folder = folder,
-            recurse = false,
+            recurse = recurse,
```

Comment:
does recursion happen within the folder or in the entire project (which is what should happen)?

Reply:
Entire project, unless a folder is specified (which it is not when `-projectWideReuse` is set)